ports/javascript/hello.py

```diff
@@ -0,0 +1,5 @@
+import js
```

Comment:
Is this file needed at all?  If so (as a simple example) it should probably go in the examples/ dir, under a new examples/javascript/ subdir.